omnibus/files/private-chef-cookbooks/private-chef/libraries/elasticsearch.rb

```diff
@@ -24,8 +28,13 @@ def self.node_memory_in_units(node, _which, unit)
   # https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html
   def self.heap_size_default(node)
     memory = node_memory_in_units(node, :total, :mb)
-    # Note: Preflight check will error if the system memory is less than 4GB.
-    [memory / 4, 1024].max
+    value = [memory / 4, 1024].max
```

Comment:
This can just be 
```
value = memory / 4
```
since the following takes care of setting the value to at least 1024

Reply:
i didnt really want to change existing code, but it passes the unit test so :yolo: